packages/client/src/__tests__/json.test.ts

```diff
@@ -89,6 +89,23 @@ describeIf(process.platform !== 'win32')('json', () => {
   })
 
   test('should be able filter json null', () => {
+    const document = makeDocument({
+      dmmf,
+      select: {
+        where: {
+          json: {
+            equals: Symbol('JsonNull'),
+          },
+        },
+      },
+      rootTypeName: 'query',
+      rootField: 'findManyOptionalUser',
+    })
+    document.validate(undefined, false, 'user', 'colorless')
+    expect(String(document)).toMatchSnapshot()
+  })
+
+  test('should not consider "JsonNull" string an enum value', () => {
     const document = makeDocument({
```

Comment:
The diff, though technically correct, is logically misleading somewhat.

What happened here is I changed `'JsonNull'` to `Symbol('JsonNull')` in the existing test (`should be able filter json null`) and added a new test (`should not consider "JsonNull" string an enum value`) that just so happens to be identical to the old one but has a new meaning.